update.js

```diff
@@ -34,6 +34,10 @@ module.exports = function update () {
     return console.error('This build should not update the lockfile. It could be a PR, not a branch build.')
   }
 
+  if (info.branchName == null) {
```

Comment:
What do you think about `(!info.branchName)` ?
so it wouldn't matter if the env variable is `''` or `undefined`